### ISSUE TITLE
IPython notebooks

### ISSUE THREAD

**Issue by brettimus** [ismayc/rticles#4]
Any interest in this format? I'm working on a project that's heavily tied to ipynb. Wondering if any current students (or alums) have had exposure to IPython / Jupyter through their work.

**Comment by ismayc** [comment]
I have lots of interest in Jupyter notebooks.  I haven't done much work with them though.  Do they have the ability to have a template file sit in the background to match the output with?  That's one reason I really like the customizability of RMarkdown with LaTeX sitting in the background, but if that also exists there I'd love to try to implement it!

**Comment by brettimus** [comment]
I'm not really familiar with what it means to have a template file sitting in the background to match with output (haven't used rmd + latex together). I do know that IPython can use mathjax, but I'm assuming that's a notch below what you're talking bout.

Wanna link me to some stuff to help me understand?? :grin:

**Comment by ismayc** [comment]
Here is the resource I used to learn how to make templates.  They have examples from a variety of different journals:  
http://rmarkdown.rstudio.com/developer_document_templates.html .

I think you will get an idea of what I mean by the template feature after seeing this.  Feel free to let me know if you'd like a different example or more details.  I'm happy to help and hopefully we can figure out a way to integrate Python in a way such as this.

**Comment by brettimus** [comment]
Awesome! I was just scanning through the source code too and got a basic feel for what was happening (https://github.com/ismayc/reedtemplates/blob/master/R/reed_thesis.R#L5)

Thanks @ismayc!

**Comment by ismayc** [comment]
No problem! The template.tex file referred to in the function call you referred to also calls the LaTeX reedthesis class file that folks use to make their thesis if they are using the LaTeX template.

If you see $text$ in the LaTeX code, that is where values are passed from the YAML header into LaTeX.

**Comment by brettimus** [comment]
Yeah this is such a nifty feature! I have to imagine the juptyer folks have made something analogous

**Comment by ismayc** [comment]
I haven't looked into the links here yet, but you might find something of value:  https://github.com/ipython/ipython/wiki/A-gallery-of-interesting-IPython-Notebooks#reproducible-academic-publications
